search-service/src/test/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeServiceTests.kt

```diff
@@ -241,4 +241,92 @@ class TemporalEntityAttributeServiceTests : TimescaleBasedTests() {
         val finalEntity = JsonUtils.toPrettyString(serializedEntity)
         assertEquals(loadSampleData("subscription.jsonld").trim(), finalEntity)
     }
+
+    @Test
+    fun `it should inject temporal numeric values in temporalValues format into an entity with two instances property`() {
```

Comment:
I'm wondering if these two new tests are a good opportunity to test the JUnit5 parameterized tests feature (https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests)?